kpi/views/v2/asset.py

```diff
@@ -343,6 +366,21 @@ def content(self, request, uid):
             'data': asset.to_ss_structure(),
         })
 
+    @action(
+        detail=True,
+        methods=['GET'],
+        renderer_classes=[renderers.JSONRenderer],
+    )
+    def counts(self, request, *args, **kwargs):
+        asset = self.get_object()
+        if not asset.has_deployment:
+            raise Http404
+        context = self.get_serializer_context()
+        if 'days' in request.query_params:
+            context['days'] = request.query_params['days']
```

Comment:
You can simplify this with `context['days'] = request.query_params.get('days', settings.MY_SETTING)`

I cannot find it but I was sure you already create a setting for a similar number of days? Maybe in another PR?